python/ray/data/tests/test_avro.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+import pytest
+
+import ray
+from ray.data.datasource import AvroDatasource
+
+
+@pytest.fixture(scope="session")
+def ray_start_regular_shared():
+    if not ray.is_initialized():
+        # Initialize Ray if it's not already running. Adjust num_cpus as necessary.
+        ray.init(num_cpus=2)
+    else:
+        # Ray is already initialized, so no further action is required.
+        pass
+    yield None
+    # No need to call ray.shutdown() here since it would terminate the existing session.
```

Comment:
I think this should already be available since it's defined in `conftest`
```suggestion
```

Reply:
removed, thanks